specs/guard-jwt.spec/src/main/scripts/io/aklivity/zilla/specs/guard/jwt/config/event.yaml

```diff
@@ -23,7 +23,8 @@ telemetry:
         events:
           - qname: test.net0
             id: guard.jwt.authorization.failed
-            message: AUTHORIZATION_FAILED user
+            name: GUARD_JWT_AUTHORIZATION_FAILED
+            message: No active session found for token identity (user).
```

Comment:
This message seems to imply we are looking for matching state in zilla and couldn't find it.

However, I think this event is logged when an attempt to authorize the user (by validating the token) fails, with no implication of state expected to be found at zilla.

Let's reword to capture the underlying behavior without implying expectation to find server-side state, agree?

Reply:
Updated please review new wording with added `reason` field.